README.md

```diff
@@ -1,10 +1,31 @@
+# Newcomers checklist
+
+Soft intro:
+
+- [ ] Introduction to the team
+- [ ] Code for Poznań introduction
+- [ ] Projects overview
+
+Tech intro:
+
+- [ ] Environment setup (Ubuntu/MacOS)
+- [ ] GitHub account
+- [ ] Invitation to the GitHub organization
+- [ ] Gitter access
+- [ ] GitHub flow
+- [ ] Zenhub
+- [ ] SSH key
+- [ ] Docker installation
+- [ ] Docker-compose installation
+- [ ] Task assignment
+
 # Structure of files
 
-Currently we have two folders - first for organization stuff, second for technical tutorials. 
+Currently we have two folders - first for organization stuff, second for technical tutorials.
 
 If you are new here, please start with reading `getting_to_know_us.md` within `organization_onboarding` folder.
```

Comment:
W sumie getting_to_know_us miało pełnić funkcję właśnie takiej checklisty, także odsyłanie z checklisty do checklisty...?

Reply:
Removed from here, so I guess this is not valid anymore?